lc2fen/detectboard/detect_board.py

```diff
@@ -1,7 +1,9 @@
+"""This is the main file of detectboard module.
+
+It detects a board on a given image using the `detect()` function.
```

Comment:
Interesting that you think that way--the way I think is that the parentheses indicate it's a function (rather than a variable) but doesn't necessarily indicate the function has no arguments.

Upon looking around on the internet, it appears that my decision is justified (even in cases where the word "function" literally follows/precedes the parentheses):
https://stackoverflow.com/questions/66498411/is-it-a-convention-in-python-documentation-to-include-parentheses-after-function